### ISSUE TITLE
Object detection

### ISSUE THREAD

**Issue by DiegoVil** [daltskin/CustomVision-TensorFlow-CSharp#2]
This is not an issue, just an idea. How I should handle the object detection from the customvision.ai model? I define this input : 
`runner.AddInput(graph["Placeholder"][0], tensor).Fetch(graph["model_outputs"][0]);`   
 but I have no idea on how to handle it.

**Comment by daltskin** [comment]
I haven't tried this with object detection.  But something along the lines of this might help?

```
runner.AddInput(graph["Placeholder"][0], tensor).Fetch("model_outputs");
var output = runner.Run();
var result = output[0];
var boxes = (float[][][][])result.GetValue(jagged: true);
```

**Comment by IamSierraCharlie** [comment]
Did you get anywhere with this?

**Comment by DiegoVil** [comment]
No. This approach did not work for me. The problem is how to get the probabilities...
